locale/en/docs/guides/domain-postmortem.md

```diff
@@ -127,16 +127,16 @@ d1.on('error', (er) => { /* handle error */ });
 d1.run(() => setTimeout(() => {
   const d2 = domain.create();
   d2.bar = 43;
-  d2.on('error', (er) => console.error(er.message, domain._stack));
+  d2.on('error', (er) => { console.error(er.message, domain._stack); });
   d2.run(() => {
     setTimeout(() => {
       setTimeout(() => {
         throw new Error('outer');
-      });
-      throw new Error('inner')
-    });
+      }, 1);
```

Comment:
This is what we use in core but I think it creates more confusion than it tries to solve here